### PR TITLE
add option to link to launch page for the latest release

### PR DESCRIPTION
**Pull request by mashehu**
As requested on Slack

### CODE REVIEW

**Review by jpfeuffer — COMMENTED** [review]
The indentation of the closing bracket looks a bit off. Or is it just the GitHub changes view?

File: public_html/launch.php [hunk 1]
```diff
@@ -61,6 +61,9 @@ function launch_pipeline_web($pipeline, $release){
     if(!array_key_exists($_GET['pipeline'], $pipelines)){
         return ["Error - Pipeline name <code>$pipeline</code> not recognised"];
     }
+    if($release == 'latest'){
+            $release = $pipelines[$pipeline][0];
+        }
```
Comment (ewels) [inline public_html/launch.php:66]:
```suggestion
    }
```

**Review by ewels — COMMENTED** [review]
This is nice, but I was kind of thinking that we could do it just by not specifying `release` in the URL. I prefer that to having a string `latest`. It's more intuitive I think, and won't mess with people using branches called `latest`..

**Comment by ewels** [comment]
Sat at a keyboard so quickly whipped up my own PR because I'm having code withdrawal symptoms.. 😆  https://github.com/nf-core/nf-co.re/pull/494

Turns out that the launch page doesn't currently work with branches, so that's not an issue after all. Maybe scope for a future PR?

**Comment by mashehu** [comment]
replaced by #494

**Pull request closed without merging**